db/migrate/20251118204141_add_privacy_radius_to_tags.rb

```
@@ -0,0 +1,8 @@
+class AddPrivacyRadiusToTags < ActiveRecord::Migration[8.0]
```

RuboCop (1.81.7)

[convention] Missing frozen string literal comment.

(Style/FrozenStringLiteralComment)